demo/node/rntuple_test.js

```diff
@@ -43,3 +43,19 @@ else {
       console.log(`OK: Field ${i}: ${field.fieldName} (${field.typeName})`);
   }
 }
+
+// Column Check
+
+if (!rntuple.builder?.columnDescriptors?.length)
+  console.error('FAILURE: No columns deserialized');
+else {
+  console.log(`OK: ${rntuple.builder.columnDescriptors.length} column(s) deserialized`);
+  for (let i = 0; i < rntuple.builder.columnDescriptors.length; ++i) {
+    const column = rntuple.builder.columnDescriptors[i];
```

Comment:
The tests for both columns and fields should be made more comprehensive (it can be in a separate PR).
E.g, for fields, verify that the field name and types are those that you expect; for columns verify that the field ID and column types are the ones you expect etc.
Does not need to be for every field and column, but at least for the first and last field they should be there. Ideally you should test all the properties of the descriptors.